src/components/badge/format.ts

```diff
@@ -0,0 +1,19 @@
+import NumAbbr from 'number-abbreviate';
+
+/**
+ * Abbreviate a number
+ *
+ * @param {number} value the number to abbreviate
+ *
+ * @returns {string} abbreviated number
+ */
+export function abbreviate(value: number): string {
+    if (typeof value !== 'number') {
+        return '';
+    }
+
+    const units = ['k', 'M', 'B', 'T'];
```

Comment:
We really should make sure these are translated, as "billion" and "trillion" are *very* different things in English compared to the rest of the world… 😣